docs/config/README.md

```diff
@@ -1132,6 +1132,13 @@ cherry_pick = "[🍒 PICKING](bold red)"
 The `git_status` module shows symbols representing the state of the repo in your
 current directory.
 
+::: tip
+
+The Git Status module is very slow in Windows directories (anything in `/mnt/c/`) when in a WSL environment.
```

Comment:
The phrasing here made me concerned that the implementation would actually be parsing the Linux path and assuming that `/mnt/c` itself indicates whether something is on the host file system (which would not be quite correct), but since that's not the case, this comment should probably make it clear that this is merely a common example. 

```suggestion
The Git Status module is very slow in Windows directories (under `/mnt/c/`, for example) when in a WSL environment.
```